ui/hooks/useMultiPolling.ts

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useRef, useState } from 'react';
+
+type UseMultiPollingOptions<PollingInput> = {
+  startPolling: (input: PollingInput) => Promise<string>;
+  stopPollingByPollingToken: (pollingToken: string) => void;
+  input: PollingInput[];
+};
+
+// Version of ui/hooks/usePolling.ts that supports multiple inputs / polling loops
+const useMultiPolling = <PollingInput>(
```

Comment:
This hook would be used by other controllers that need multiple active polling loops like TokenListController